library/src/updater.rs

```diff
@@ -312,8 +162,39 @@ fn update_internal(config: &ResolvedConfig) -> anyhow::Result<UpdateStatus> {
     debug!("app_dir: {:?}", app_dir);
     let base_r = open_base_lib(&app_dir, "libapp.so")?;
 
+    inflate(&download_path, base_r, &output_path)
+}
+
+#[cfg(not(any(target_os = "android", test)))]
```

Comment:
Thoughts on making this explicit that we're targeting iOS (as opposed to "not Android"). Not sure how doable that is since we're also making sure this isn't a test cfg

Reply:
In this case iOS *should* use this function, just doesn't yet.  That's why iOS patch files are MB instead of bytes.